### DISCUSSION TITLE
Support for Exchange Online via Azure APP

### DISCUSSION THREAD

**Discussion by Ragchel** [pelican/panel#563] [Ideas]
Good afternoon,

Microsoft made a change and deemed SMTP as legacy and now tries to enforce apps to allow third party apps to send mail, I would like to see the possibility to use an App to set up mail from Exchange Online without getting scolded by Microsoft for not being best practice.

I am open to give access to test in an existing environment so that a new environment doesn't have to be set up as this is probably a very niche request.

If more information is required do let me know.

**Comment by lancepioch** [comment]
Sorry, not planned.
